style.css

```diff
@@ -910,3 +917,1217 @@ footer #contact-form .contact-details p {
+    display: flex;
+    list-style: none;
+    flex-direction: row;
+    margin-left: 10px;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    gap: 1rem;
+    margin-top: 2rem;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .HTML {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 10px;
+    width: 53px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .Ruby-on-rails {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 14px;
+    width: 110px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .CSS {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 10px;
+    width: 45px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .JavaScript {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 14px;
+    width: 94px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  #About {
+    display: flex;
+    margin-top: 6rem;
+    background-image: url("Images/self-desktop1.png");
+    background-size: cover;
+    width: 100%;
+    height: auto;
+    padding-bottom: 10rem;
+  }
+
+  #About .Aboutcontent {
+    padding-top: 8rem;
+    margin-left: 5rem;
+  }
+
+  #About .Aboutcontent h3 {
+    width: 100%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  #About .Aboutcontent p {
+    width: 447px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: #344563;
+    padding-top: 1.2rem;
+    padding-bottom: 1rem;
+  }
+
+  #About .Aboutcontent a {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    color: #7f8cff;
+    text-decoration: none;
+    margin-top: -0.2rem;
+  }
+
+  #About .iconsAbout ul {
+    display: flex;
+    list-style: none;
+  }
+
+  #About .iconsAbout a {
+    padding-left: 1.5rem;
+  }
+
+  #About .info {
+    margin-top: 9.5rem;
+    margin-right: 5rem;
+  }
```

Comment:
- [ ] The list of technologies in your design looks smaller than it is in the actual design template. Kindly fix this. You can consider adding a width of 100% to this selector and it will fix this.
### Your design
![image](https://user-images.githubusercontent.com/94551484/236443772-c1c42574-36b8-4ec5-b0bf-a585a2d4ba65.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/236443852-83715d69-9faf-4099-bce6-1d1501271dd2.png)